ogusa/model_solution_parameters.json

```diff
@@ -0,0 +1,152 @@
+{
+    "MINIMIZER_TOL": {
+        "long_name": "Tolerance used in root finders to solve household FOCs",
+        "description": "Tolerance used in root finders to solve household FOCs",
+        "notes": "",
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [1e-14],
+        "range": {"min": 1e-16, "max": 0.001},
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop",
+        "compatible_data": {"puf": true, "cps": true}
+    },
+    "MINIMIZER_OPTIONS": {
+        "long_name": "Options used in root finders to solve household FOCs",
+        "description": "Options used in root finders to solve household FOCs",
+        "notes": "",
+        "col_var": "",
+        "col_label": "",
+        "boolean_value": false,
+        "integer_value": false,
+        "value": [None],
+        "range": {"min": , "max": },
+        "out_of_range_minmsg": "",
+        "out_of_range_maxmsg": "",
+        "out_of_range_action": "stop",
+        "compatible_data": {"puf": true, "cps": true}
+    },
```

Comment:
@jdebacker how is `MINIMIZER_OPTIONS` used? I did a search in the code base and can see where it's set but not where it is used.

Reply:
@hdoupe You are right - it is not used.  I'll drop this from the JSON.

Reply:
Ok, sounds good.